docs/libcurl/opts/CURLOPT_HTTP_TRAILER_DATA.3

```diff
@@ -0,0 +1,58 @@
+.\" **************************************************************************
+.\" *                                  _   _ ____  _
+.\" *  Project                     ___| | | |  _ \| |
+.\" *                             / __| | | | |_) | |
+.\" *                            | (__| |_| |  _ <| |___
+.\" *                             \___|\___/|_| \_\_____|
+.\" *
+.\" * Copyright (C) 1998 - 2017, Daniel Stenberg, <daniel@haxx.se>, et al.
+.\" *
+.\" * This software is licensed as described in the file COPYING, which
+.\" * you should have received as part of this distribution. The terms
+.\" * are also available at https://curl.haxx.se/docs/copyright.html.
+.\" *
+.\" * You may opt to use, copy, modify, merge, publish, distribute and/or sell
+.\" * copies of the Software, and permit persons to whom the Software is
+.\" * furnished to do so, under the terms of the COPYING file.
+.\" *
+.\" * This software is distributed on an "AS IS" basis, WITHOUT WARRANTY OF ANY
+.\" * KIND, either express or implied.
+.\" *
+.\" **************************************************************************
+.\"
+.TH CURLOPT_HTTP_TRAILER_DATA 3 "14 Aug 2018" "libcurl 7.??.?" "curl_easy_setopt options"
+
+.SH NAME:
+CURLOPT_HTTP_TRAILER_DATA \- Set user data to be passed on to the trailing headers callback 
+if one set.
+		
+.SH SYNOPSIS:
+#include <curl.h>
+
+CURLcode curl_easy_setopt(CURL *handle, CURLOPT_HTTP_TRAILER_DATA, void *userdata);
+
+.SH DESCRIPTION:
+Data pointer to be passed to the HTTP trailer callback function.
+
+.SH DEFAULT:
+NULL
+
+.SH PROTOCOLS:
+HTTP
+
+.SH EXAMPLE:
+.nf
+// Assuming we have a CURL handle in the hndl variable.
```

Comment:
Since we discourage `//` comments everywhere (for C89 portability), we probably should use them in the man page example either...